utils/addresses.ts

```diff
@@ -8,14 +8,26 @@ interface IfcOneNetworkAddresses {
   USDC?: string;
   WMATIC?: string;
 }
-interface IfcAddresses {
-  celo: IfcOneNetworkAddresses;
-  alfajores: IfcOneNetworkAddresses;
-  polygon: IfcOneNetworkAddresses;
-  mumbai: IfcOneNetworkAddresses;
+interface InterfaceAddresses {
```

Comment:
Same comment as above:
```suggestion
interface IAddresses {
```